robyn/processpool.py

```diff
@@ -18,7 +18,8 @@ def run_processes(
     directories: List[Directory],
     request_headers: List[Header],
     routes: List[Route],
-    middlewares: List[MiddlewareRoute],
+    global_middlewares: Dict[MiddlewareType, List[FunctionInfo]],
```

Comment:
@AntoineRR , let us abstract this into `List[GlobalMiddleWareRoute]`, this will make the function definition consistent.

Reply:
Agreed, updated it.